src/sql/workbench/browser/actions.ts

```diff
@@ -88,3 +89,20 @@ export class ConfigureDashboardAction extends Task {
 		accessor.get(IOpenerService).open(URI.parse(ConfigureDashboardAction.configHelpUri));
 	}
 }
+
+export class ClearSavedAccountsAction extends Task {
+	public static readonly ID = 'clearSavedAccounts';
+	public static readonly LABEL = nls.localize('clearSavedAccounts', "Clear all saved accounts");
+
+	constructor() {
+		super({
+			id: ClearSavedAccountsAction.ID,
+			title: ClearSavedAccountsAction.LABEL,
+			iconPath: undefined
+		});
+	}
+
+	async runTask(accessor: ServicesAccessor): Promise<void> {
+		accessor.get(IAccountManagementService).removeAccounts();
```

Comment:
nit: await this so we don't have a dangling promise (now of course whether this failing is going to do anything is a different story...)

Reply:
Good catch, thank you!